packages/amplication-client/src/OutdatedVersionAlerts/OutdatedVersionAlertListDataColumns.tsx

```diff
@@ -116,7 +116,12 @@ export const COLUMNS: DataGridColumn<OutdatedVersionAlert>[] = [
     resizable: false,
     sortable: false,
     renderCell: (props) => {
-      return <AlertLink alert={props.row} />;
+      return (
+        <AlertLink
+          alert={props.row}
+          projectId={props.row.resource?.projectId}
```

Comment:
since project is already part of the resource and we send the entire row - we can remove the extra param.